exercicio1/scripts/exercicio1.ts

```diff
@@ -0,0 +1,35 @@
+const showWord = document.querySelector("#showWord") as HTMLElement;
+const vowels = document.querySelector("#vowels") as HTMLElement;
+const submitWord = document.querySelector("#submitWord") as HTMLButtonElement
+const inputWord = document.querySelector("#word") as HTMLInputElement
+
+
+countVowels("JOGADOR");
+
+function countVowels(verifyWord: string):number {
+    const VOGAIS: string[] = ["A","E","I","O","U"];
+    let word: string = verifyWord.normalize("NFD").replace(/[\u0300-\u036f]/g, "").toUpperCase()
+    let count: number = 0;
+    
+    for(let i:number = 0; i<word.length; i++){
```

Comment:
O que acha de refatorarmos para uma solução  mais funcional, no sentido do paradigma mesmo? As vezes quando utilizamos o for, com variáveis não muito legíveis, o código fica dificil de dar manutenção, e certos métodos no javascript possuem um pézinho no paradigma funcional, que é muitas coisas mas diferente do paradigma imperativo que foca em 'COMO FAZER', o  funcional foca em "O QUE" fazer e isso se traduz na legiblidade.

Poderia utilizar o .reduce aqui do js para gerar o valor da variável count.

- Documentção do Reduce: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

basicamente ele tranforma uma lista em um unico valor ao final, perfeito para o nosso caso de uso.